bin/schema_generator.sh

```diff
@@ -134,10 +134,6 @@ function main() {
             continue
         fi
```

Comment:
`metadata/sources` is in the `disallowlist`, so I expect that would exclude it. Is this case still needed?

I think we should probably add a `metadata/.*` line to `disallowlist` and remove all the specific cases, as MPS itself documents that the `metadata` directory is not a real namespace, so I never expect us to add a real ping type there.

Reply:
What the disallowlist does is prevent checking in schemas on commit. What this special case does is prevent BQ schema generation in the first place, since processing this AWS-specific schema results in an exception.

Metadata contains the schemas for the decoded, raw, and error tables, which are special cases handled by the ops logic separately.

Reply:
@whd this question is up your alley